docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+123456789012345678version: "2.2"
```

Comment:
any particular reason this exists?

Reply:
i was fixing my tokens and miss typed. didnt not realise the mistake straight way